internal/assertions/bell_assertion.go

```diff
@@ -0,0 +1,43 @@
+package assertions
+
+import (
+	"time"
+)
+
+const defaultBellTimeout = 100 * time.Millisecond
```

Comment:
Let's see if we can remove `defaultBellTimeout`, this can be confusing to users since it isn't the actual "timeout" that we use when listening for a bell - that comes from AssertionCollection